indigo_app/views/works.py

```diff
@@ -566,11 +566,15 @@ def form_valid(self, form):
         works = None
         locality_code = self.locality.code if self.locality else None
         bulk_creator = plugins.for_locale('bulk-creator', self.country.code, None, locality_code)
+        extra_properties_getter = plugins.for_locale('extra-properties', self.country.code, None, locality_code)
```

Comment:
What if we simplify this and simply have a list of extra properties on the BulkCreator object? We already have plugin support for that.

Reply:
wonderful idea!